Dockerfile

```diff
@@ -0,0 +1,129 @@
+FROM ubuntu:16.04
+
+
+LABEL authors="rickard.hammaren@scilifelab.se, phil.ewels@scilifelab.se" \
+    description="Docker image containing all requirements for NGI-RNAfusion pipeline"
+
+#Fusioncatcher 
+RUN apt-get -y update
+RUN apt-get -y install \
+    build-essential\
+    libncurses5-dev \
+    default-jdk \
+    gawk \
+    gcc \
+    g++ \
+    bzip2 \
+    cmake \
+    automake \
+    gzip \
+    zip \
+    unzip \
+    zlib1g-dev \
+    zlib1g \
+    wget \
+    curl \
+    pigz \
+    tar \
+    parallel \
+    libtbb-dev \
+    libtbb2 \
+    python \
+    python-dev \
+    python-numpy \
+    python-biopython \
+    python-xlrd \
+    python-openpyxl
+
+RUN wget http://sf.net/projects/fusioncatcher/files/bootstrap.py -O bootstrap.py && python bootstrap.py -t --download -y && rm bootstrap.py
+
+
+
+#### STAR-Fusion START 
+RUN apt-get update && apt-get install -y gcc g++ perl python automake make \
```

Comment:
These `apt-get` installations can be merged with the other `apt-get` installation command above (most are already installed in that list anyway).